lib/class-vip-request-block.php

```diff
@@ -36,21 +38,31 @@ public static function ip( string $value ) {
 			return false;
 		}
 
+		// In case this is an IPv6 address and PHP is compiled without the IPV6 support, make sure `false`'s won't match.
+		if ( false === $ip ) {
```

Comment:
Should this be more explicit to include a check whether PHP was compiled without IPV6 support? As I'm reading the docs it says "false if a syntactically invalid ip is given",  so we kind of side-stepping the intent of that `false` .

Reply:
@rinatkhaziev we validate the IP here: https://github.com/Automattic/vip-go-mu-plugins/pull/2756/files#diff-2aaa18fe2bb071a15fb049eb6bc4dcf92755b5f1baa6171d13e26330f629b38fL30

If it is not valid, we log a warning and exit.